.github/workflows/main.yml

```diff
@@ -16,72 +16,331 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  test:
-    name: Test
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # 4.2.2
```

Comment:
Ah sorry -- very used to doing this in repos where Dependabot is turned *on* -- can we turn it on? It can actually recognize and update these tags (even without the version hint)

Reply:
Ah OK! make sense -- I'll revert the pinned versions real quick!